CHANGELOG.md

```diff
@@ -35,6 +35,16 @@ All notable changes to this project will be documented in this file.
     * #### Removed
 * ### `nidcpower` (NI-DCPower)
     * #### Added
+        * API parity with NI-DCPower 21.8.0.
```

Comment:
just to confirm... does this PR complete the full parity or are there more things missing coming in future PRs?

Reply:
Nope, this PR does not complete the full parity. There are more things that are coming in future PRs. I have updated the CHANGELOG.md in my latest commit to clarify that this PR only achieves partial API parity.